packages/pigeon/lib/objc_generator.dart

```diff
@@ -1488,21 +1488,11 @@ String _makeObjcSignature({
 void generateObjcHeader(ObjcOptions options, Root root, Indent indent) {}
 
 String _listGetter(String list, NamedType field, int index, String? prefix) {
```

Comment:
This is only used in one place; now that it's one line we should just inline it at the call site. The string itself is pretty self-explanatory.